backend/src/test/kotlin/app/ehrenamtskarte/backend/IntegrationTestConfig.kt

```diff
@@ -3,10 +3,12 @@ package app.ehrenamtskarte.backend
 import app.ehrenamtskarte.backend.config.BackendConfiguration
 import org.springframework.boot.test.context.TestConfiguration
 import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Primary
 
 @TestConfiguration
 class IntegrationTestConfig {
     @Bean
+    @Primary
     fun backendConfiguration(): BackendConfiguration {
```

Comment:
🙃  This is optional, since you didn't touch it
But shouldn't this throw a `FileNotFoundException` ?
This can also be inlined

```suggestion
    fun backendConfiguration(): BackendConfiguration = BackendConfiguration.load(ClassLoader.getSystemResource("config.test.yml")
        ?: throw FileNotFoundException("src/test/resources/config.test.yml")
    )
```
And it could be inlined also